plugins/generator-1.20.1/forge-1.20.1/templates/json/item_model_blocking.json.ftl

```diff
@@ -0,0 +1,49 @@
+{
+<#if data.blockingRenderType == 0>
+    "parent": "item/handheld",
+    "textures": {
+        "layer0": "${modid}:item/${data.texture}"
+    },
+    "display": {
+        "thirdperson_righthand": {
+            "rotation": [ 45, -35, 0 ]
+        },
+        "thirdperson_lefthand": {
+            "rotation": [ 45, -35, 0 ]
+        },
+        "firstperson_righthand": {
+            "rotation": [ 0, 0, 5 ],
+            "translation": [ -5, 2, -1 ]
+        },
+        "firstperson_lefthand": {
+            "rotation": [ 0, 0, 5 ],
+            "translation": [ -5, 2, -1 ]
+        }
+    }
+}
+<#elseif data.blockingRenderType == 1>
+    "parent": "${modid}:custom/${data.blockingModelName.split(":")[0]}",
+    "textures": {
+        <@textures data.getBlockingTextureMap()/>
+        "particle": "${modid}:items/${data.texture}"

```

Comment:
This is the only place where I see items being used.

How sure can we be this is tested given this will not work?

Reply:
✔️ Fixed